.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,91 @@
+version: '3.8'
```

Comment:
We now have versions of this file scattered all over the repo. Should we consolidate them all into one place (one source of the truth)?

Reply:
Will work on this, good idea.

Reply:
Consolidated. Noticing further duplication of doc content that could use revisiting but that's outside scope of this PR.